cli/src/action/circuit/api.rs

```diff
@@ -301,6 +301,61 @@ pub struct ProposalSlice {
     pub requester_node_id: String,
 }
 
+impl fmt::Display for ProposalSlice {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut display_string = format!(
+            "Proposal to create:: {}\n    Management Type: {}\n",
+            self.circuit_id, self.circuit.circuit_management_type
+        );
+
+        for member in self.circuit.members.iter() {
+            display_string += &format!("\n    {} ({})\n", member.node_id, member.endpoint);
+            if member.node_id == self.requester_node_id {
+                display_string += &"        Vote: ACCEPT (implied as requester):\n".to_string();
+                display_string += &format!("            {}\n", self.requester);
+            } else {
+                let mut vote_string = "        Vote: PENDING".to_string();
+                for vote in self.votes.iter() {
+                    if vote.voter_node_id == member.node_id {
+                        vote_string =
+                            format!("        Vote: ACCEPT\n             {}", vote.public_key)
+                    }
+                }
+                display_string += &format!("{}\n", vote_string);
+            }
+            for service in self.circuit.roster.iter() {
+                if service.allowed_nodes.contains(&member.node_id) {
+                    display_string += &format!(
+                        "        {} ({}): {}\n",
+                        "Service", service.service_type, service.service_id
```

Comment:
"Service" can be in the format string itself

Reply:
fixed